news/1632.misc

```diff
@@ -0,0 +1,2 @@
+Update buildout and requirements to Plone-6.0.4
```

Comment:
We use `@github_username` format now.

```suggestion
Update buildout and requirements to Plone-6.0.4. @1letter
```

Reply:
Also `.misc` is not a valid file extension. See https://github.com/plone/plone.restapi/blob/master/pyproject.toml for valid extensions. This should be `1632.bugfix`.

Reply:
@1letter @stevepiercy I'd suggest using the "internal" file extension here.

Reply:
You are correct. `.internal` is more appropriate.